src/compiler/checker.ts

```diff
@@ -19637,6 +19263,7 @@ namespace ts {
             const isTaggedTemplate = node.kind === SyntaxKind.TaggedTemplateExpression;
             const isDecorator = node.kind === SyntaxKind.Decorator;
             const isJsxOpeningOrSelfClosingElement = isJsxOpeningLikeElement(node);
+            const reportErrors = !candidatesOutArray;

```

Comment:
I know it's redundant, but it might be more readable to add a reportErrors parameter.

Reply:
It already has an optional `fallbackError` parameter that I'd then need to start providing (as `undefined`) at almost every callsite.